packages/terra-navigation-prompt/src/NavigationPromptCheckpoint.jsx

```diff
@@ -133,10 +133,13 @@ class NavigationPromptCheckpoint extends React.Component {
       this.setState({
         notificationDialogProps: {
           title: showDialogOptions.title,
-          message: showDialogOptions.message,
```

Comment:
I think you still need this line to remain passive.

Reply:
Otherwise, your usage of message below for `startMessage` will never work.

Reply:
Yep your right. Fixed here https://github.com/cerner/terra-framework/pull/899/commits/483a9f77e989716d5f30da81ea6e42cb7d6ee213